### ISSUE TITLE
Collapsing rows when grouped?

### ISSUE THREAD

**Issue by cliren** [zippyui/react-datagrid#17]
I haven't seen an example to collapse rows, is it simple enough to implement it when using row grouping?

**Comment by zippyui-react** [comment]
The feature is not implemented, but shouldn't be too difficult.

We already have a treegrid that we're going to publish soon, but don't have group collapsing. The treegrid is built on top of the datagrid.

**Comment by NogsMPLS** [comment]
+1 for ability to expand/collapse a group of rows

**Comment by kaanyurukcu** [comment]
Any progress on this?

**Comment by phkavitha** [comment]
Row expansion would be a very nice feature.

**Comment by txm** [comment]
+1 :)

**Comment by amorri40** [comment]
Before I start looking into implementing this has anyone made a start on this feature?

**Comment by phkavitha** [comment]
@amorri40 , I started working on it but, once the v3 branch became very active I paused it for a while. I might start the development once the v3 branch is merged to master. I too need this feature very badly for my project. Are you going to work on master or v3 branch?

**Comment by radubrehar** [comment]
v2 is going to be parked - we're actively pushing the v3 onwards.

**Comment by amorri40** [comment]
I was going to work on master as I couldn't get the v3 branch to work, it seems to be very different than the current implementation. Ideally it would be good to get v3 up and running and make my changes there, but since we are using v2 i'll start there.

**Comment by phkavitha** [comment]
@amorri40 , I'm also interested in this feature and would like to contribute. Have you forked the repo? I can share the changes which I did for row collapse.
